vsintegration/src/vs/FsPkgs/FSharp.Project/FS/MSBuildUtilities.fs

```diff
@@ -108,6 +159,24 @@ type internal MSBuildUtilities() =
                     if x.[i] <> y.[i] then
                         r <- false
                 r
+
+        let relativePath path = PackageUtilities.MakeRelativeIfRooted(path, projectNode.BaseURI)
+
+        let checkAlreadyRenderedFolder visitedDirs (bi : ProjectItemElement, path) =
+            let dir = Path.GetDirectoryName(path) + "\\"
+
+            match MSBuildHelpers.IsAlreadyRenderedFolder visitedDirs dir with
+            | Some _x when throwIfCannotRender ->
+                let msg = String.Format(FSharpSR.GetString(FSharpSR.ProjectRenderFolderMultiple), projectNode.ProjectFile, bi.Include)
+                raise (InvalidOperationException(msg))
+            | Some _ | None ->
+                dir :: visitedDirs
+
+        EnumerateItems(big)
+        |> Seq.map (fun bi -> bi, GetUnescapedUnevaluatedInclude(bi) |> relativePath)
```

Comment:
The `relativePath` function, and first line of `checkAlreadyRenderedFolder` are together doing the same thing as existing function `ComputeFolder`.

Recommend you just call that function here, and remove `relativePath` function and first line of `checkAlreadyRenderedFolder`